packages/pubsub/system-test/pubsub.js

```diff
@@ -144,11 +139,9 @@ describe('pubsub', function() {
     it('should allow manual paging', function(done) {
       pubsub.getTopics({
         pageSize: TOPIC_NAMES.length - 1
-      }, function(err, topics, nextQuery) {
+      }, function(err, topics) {
```

Comment:
We still want `autoPaginate` in our library, which internally sets the gax option.